Makefile

```diff
@@ -35,9 +36,17 @@ install:
 ### Tools & dependencies
 
 check_tools:
-	@# https://stackoverflow.com/a/25668869
-	@echo "Found tools: $(foreach tool,$(GOTOOLS_CHECK),\
-        $(if $(shell which $(tool)),$(tool),$(error "No $(tool) in PATH")))"
+	@echo "Checking go availability"
+	@test -x $(shell which go)
+	@echo "Checking go version $(GO_VERSION) >= $(GO_MIN_VERSION)"
+	@test $(shell echo $(GO_VERSION) | grep -o '^[[:digit:]]\+' ) -ge $(shell echo $(GO_MIN_VERSION) | grep -o '^[[:digit:]]\+' )
```

Comment:
Do we really need this? Can't we just say that required go version is 1.9 in README?

Reply:
I'd vote against this PR. It brings additional complexity and I am not sure we need it.

Reply:
I'm trying to minimize the amount of documentation required for the build process. Documentation can get obsolete but automation is tested with every commit.

Reply:
A better version was created. It's pull requested at #1042